department-of-veterans-affairs/caseflow
# Fix issue with notes not being captured on add hearing day modal

Resolves #14394

### Description

  - Update `hearingSchedule` reducer to handle the `SET_NOTES` event
### Testing Plan
1. Login as a Hearing Coordinator
2. Click `Add Hearing Date`
3. Fill out form, and notes field
4. Confirm
5. Go to new hearing day, and confirm the notes section is there
